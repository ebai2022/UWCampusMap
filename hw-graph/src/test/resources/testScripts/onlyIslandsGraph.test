# Test graph with islands behavior

# Create a graph
CreateGraph A

# Add nodes
AddNode A e1
AddNode A e2

# Check if it is an island
ListChildren A e1
ListChildren A e2