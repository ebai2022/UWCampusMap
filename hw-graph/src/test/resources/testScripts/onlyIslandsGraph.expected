# Test graph with islands behavior

# Create a graph
created graph A

# Add nodes
added node e1 to A
added node e2 to A

# Check if it is an island
the children of e1 in A are:
the children of e2 in A are: